dubbo-rpc/dubbo-rpc-dubbo/src/test/java/org/apache/dubbo/rpc/protocol/dubbo/telnet/InvokerTelnetHandlerTest.java

```diff
@@ -122,7 +124,7 @@ public void testInvokeByPassingEnumValue() throws RemotingException {
 
     @SuppressWarnings("unchecked")
     @Test
-    public void testComplexParamWithoutSpecifyParamType() throws RemotingException {
+    public void testOverriddenMethodWithSpecifyParamType() throws RemotingException {
         mockChannel = mock(Channel.class);
         given(mockChannel.getAttribute("telnet.service")).willReturn("org.apache.dubbo.rpc.protocol.dubbo.support.DemoService");
```

Comment:
can be rewritten as 
`given(mockChannel.getAttribute("telnet.service")).willReturn(DemoService.class.getName());`